ql/src/test/org/apache/hadoop/hive/ql/udf/generic/TestGenericUDFDateFormat.java

```diff
@@ -176,7 +176,6 @@ public void testWrongFmt() throws HiveException {
 
```

Comment:
Add test for date older then 1900 with timestamp.

Reply:
Done